codeworld-api/codeworld-api.cabal

```diff
@@ -61,7 +61,7 @@ Library
                        mtl                  >= 2.2.1 && < 2.3,
                        random               >= 1.1   && < 1.2,
                        ref-tf               >= 0.4   && < 0.5,
-                       reflex               >= 0.6.3 && < 0.7,
+                       reflex               >= 0.6.3,
```

Comment:
I believe we need upper bounds on versions to upload to hackage.  So feel free to relax bounds, but let's leave them in place for codeworld-api.  Other packages don't need upper bounds, since they don't get released to hackage.